site/src/pages/extrinsic.js

```diff
@@ -98,14 +98,7 @@ function Extrinsic() {
   return (
     <Layout>
       <BreadCrumb
-        data={[
-          { name: "Extrinsics", path: "/extrinsics" },
-          {
-            name:
-              `${extrinsic?.indexer?.blockHeight}-${extrinsic?.indexer?.extrinsicIndex}` ??
-              "...",
-          },
-        ]}
+        data={[{ name: "Extrinsics", path: "/extrinsics" }, { name: id }]}
```

Comment:
what if `id` is hash ? we want the `blockHeight-index` format always

Reply:
url might be like `/#/extrinsic/0xd3416b3db18cf09b32c32cb3bc136b7558d360d8f22678876838d54ac7f6ca37`